website/app/error.tsx

```
@@ -0,0 +1,30 @@
+'use client'
+
+export default function Error({
```

Biome (2.3.14)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)